js/src/dropdown.js

```diff
@@ -192,10 +187,10 @@ const Dropdown = (($) => {
           continue
         }
 
-        // remove backdrop only if the dropdown menu will be hidden
-        const backdrop = $(parent).find(Selector.BACKDROP)[0]
-        if (backdrop) {
-          backdrop.parentNode.removeChild(backdrop)
+        // if this is a touch-enabled device we remove the extra
+        // empty mouseover listeners we added for iOS support
+        if ('ontouchstart' in document.documentElement) {
+          $('body').children().off('mouseover', $.noop)
```

Comment:
UNACCEPTABLE. no, seriously, no worries. i'll do a quick patch and still use the second parameter as selector, but as `null` (to make absolutely clear i'm wanting to bind directly to the child elements, and not do event delegation or some such)